metaflow/plugins/env_escape/stub.py

```diff
@@ -276,9 +276,21 @@ def __call__(cls, *args, **kwargs):
         if len(args) > 0 and id(args[0]) == id(cls.___class_connection___):
             return super(MetaWithConnection, cls).__call__(*args, **kwargs)
         else:
-            return cls.___class_connection___.stub_request(
-                None, OP_INIT, cls.___class_remote_class_name___, *args, **kwargs
-            )
+            if hasattr(cls, "__init_overriden__"):
```

Comment:
This covers the overrides on the client side. We need to do something similar on the server side to cover overrides there as well.